packages/grid/src/vaadin-grid-filter.js

```diff
@@ -34,97 +32,18 @@ import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
  * };
  * ```
  *
- * @fires {CustomEvent} value-changed - Fired when the `value` property changes.
```

Comment:
Is there any reason why this was removed? It's still present in `.d.ts` file.

Reply:
Moved it back here from the mixin.